rasa/core/training/structures.py

```diff
@@ -92,6 +92,10 @@ def __repr__(self):
 
 
 class StoryStep(object):
+    """A StoryStep is a linear section of a story block that contains
+        no branching (either due to checkpoints or combining multiple
+        intents with "OR"s)."""
```

Comment:
I think, it's a bit unclear, maybe rephrase it that it is the story block between checkpoints

Reply:
I've updated it.